src/Metalama/Metalama.Compiler.CodeAnalysis/MetalamaCompilerMessageProvider.cs

```diff
@@ -270,8 +271,9 @@ WRN_AnalyzerAssembliesRedirected or
                 ERR_ManyTransformersOfSameName => "The project contains several transformers named '{0}': {1}.",
                 ERR_TransformerNotUnique => "There are several transformers named '{0}': {1}.",
                 WRN_AnalyzerAssembliesRedirected => "Analyzer assemblies for this project reference Roslyn version {0}, which is newer than what is supported by the current version of Metalama ({1}). Some analyzer assemblies were downgraded to the latest supported version.",
-                WRN_AnalyzerAssemblyCantRedirect => "Analyzer assembly '{0}' was disabled, because it references Roslyn version {1}, which is newer than what is supported by the current version of Metalama ({2}).",
+                WRN_AnalyzerAssemblyCantRedirect => "Analyzer assembly '{0}' was disabled, because it references Roslyn version {1}, which is newer than what is supported by the current version of Metalama ({2}). Consider updating Metalama to a newer version, if one is available.",
```

Comment:
"""The analyzer assembly '{0}' was disabled because it references Roslyn version {1}, which is newer than the version supported by the current version of Metalama ({2}). Consider one of the following remedies. (1) Update Metalama to a newer version, if one is available.  (2) Set the version of the .NET SDK to {3} in global.json with and limit roll-forwarding the SDK to the latest patch: {{ "sdk": {{ "version": "{3}", "rollForward": "latestPatch" }} }}""". (3) If this analyzer assembly is not essential, disable the LAMA0621 warning in your project file.